requirements.txt

```diff
@@ -1,23 +1,51 @@
+alembic==0.8.10
 appnope==0.1.0
+astroid==2.1.0
 autopep8==1.3.5
+Babel==2.6.0
 backcall==0.1.0
 beautifulsoup4==4.6.1
 bleach==2.1.3
 bs4==0.0.1
 certifi==2018.4.16
 chardet==3.0.4
+Click==7.0
+colorama==0.4.1
+configparser==3.5.0
+croniter==0.3.26
 cycler==0.10.0
 decorator==4.3.0
+defusedxml==0.5.0
+dill==0.2.8.2
+docopt==0.6.2
+docutils==0.14
 entrypoints==0.2.3
+Flask-Admin==1.4.1
```

Comment:
do we need all of the new dependencies?

Reply:
Absolutely not, please check my commit msg for the same! Will remove all the dependencies and send it across.

Reply:
https://github.com/raghothams/fontastic/pull/3/commits/a5e83e1181396a13767a23a695f110d74b4b16c3